code/modules/mob/living/simple_animal/friendly/nian_caterpillar.dm

```diff
@@ -64,6 +64,9 @@
 		to_chat(src, "<span class='warning'>You need to binge on food in order to have the energy to evolve...</span>")
 		return
 
+	if(master_commander)
+		to_chat(src, "<span class='userdanger'>As you evolve, your mind grows out of it's restraints. You are no longer loyal to [master_commander]!</span>")
```

Comment:
Same as above, it's to its'

Reply:
see my above comment

Reply:
Both conversations good to resolve @Qwertytoforty